guides/william/DNSNotes.md

```diff
@@ -0,0 +1,164 @@
+(Ubuntu) DNS Notes
+======
+### Contact
+- Slack: @manwthglasses on [slack](wcscusf.slack.com)
+- Email: wmurphy1@mail.usf.edu
+
+### Table 
+1. [Prerequisites](#prerequisites)
+2. [Summary](#summary)
+3. [Background](#background)
+4. [Installation](#installation) 
+5. [Configuration](#configuration)
+6. [Helpful links](#helpful-links)
+
+### Prerequisites
+1. [Download virtualization program](https://www.virtualbox.org/wiki/Downloads)
+2. Download ISO images
+3. [Setup virtual environment](https://github.com/Nanjuan/WCSC-Blue-Team/blob/master/Network%20Adapter%20Information.md) 

```

Comment:
When this is merged in along with Nestor's and Sharon's guides make another pull request with the new links. For now this is good, just a note for the future.